Source/p_inter.c

```diff
@@ -837,6 +837,11 @@ void P_DamageMobj(mobj_t *target,mobj_t *inflictor, mobj_t *source, int damage)
         }
 
       player->health -= damage;       // mirror mobj health here for Dave
+      // BUDDHA cheat
```

Comment:
So, the BUDDHA cheat prevents the player from dying. Am I wrong, I think I remembered that this cheat would even slowly recover the cheating player's health over time?

Reply:
No, I checked in GZDoom and health is not recovering. This cheat is about health management.